numpy/core/src/multiarray/array_method.c

```diff
@@ -865,18 +860,13 @@ PyUFunc_DefaultMaskedInnerLoopSelector(PyUFuncObject *ufunc,
     data->unmasked_stridedloop = NULL;
     data->nargs = nargs;
 
-    /* Get the unmasked ufunc inner loop */
-    retcode = ufunc->legacy_inner_loop_selector(ufunc, dtypes,
-            &data->unmasked_stridedloop, &data->innerloopdata,
-            out_needs_api);
-    if (retcode < 0) {
-        PyArray_free(data);
-        return retcode;
+    if (context->method->get_strided_loop(context,
+            aligned, 0, fixed_strides,
+            &data->unmasked_stridedloop, &data->unmasked_auxdata, flags) < 0) {
+        PyMem_Free(data);
```

Comment:
return value got lost

Reply:
This one is fine I think, I don't see `get_strided_loop` getting a meaningful return value anytime soon?